### PR TITLE
Add visualizing metrics using Grafana Tutorial

### PR DESCRIPTION
**Pull request by yolossn**
To Do:
- [x] Change the video links, right now it is in my personal account.

### CODE REVIEW

File: content/docs/tutorials/visualizing_metrics_using_grafana.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Once `Grafana` is installed and run, navigate to `http://localhost:3000` in your browser. Use the default credentials, username as `admin` and password as `admin` to log in and setup new credentials.
+
+
+## Adding Prometheus as a Data Source in Grafana.
+Let's add a datasource to Grafana by clicking on the gear icon in the side bar and select `Data Sources`
+> ⚙ > Data Sources
+
+In the Data Sources screen you can see that `Grafana` supports multiple data sources like `Graphite`, `PostgreSQL` etc. Select `Prometheus` to set it up.
+
+Enter the URL as `http://localhost:9090` under the HTTP section and click on `Save and Test`.
```
Comment (hdost) [inline content/docs/tutorials/visualizing_metrics_using_grafana.md:27]:
```suggestion
Enter the URL as [http://localhost:9090](http://localhost:9090) under the HTTP section and click on `Save and Test`.
```

**Review by hdost — CHANGES_REQUESTED** [review]
Mostly looks good 👍🏼 just a couple of minor suggestions.

**Comment by yolossn** [comment]
Made the changes according to the suggestions, Once the files are uploaded ill change the links.

**Comment by hdost** [comment]
> Made the changes according to the suggestions, Once the files are uploaded ill change the links.

Who needs to upload the videos. Is that something @juliusv can do? If so where do you have the sources for him ti do so?

**Comment by yolossn** [comment]
@RichiH Uploaded them, have updated the links. @hdost PTAL.

File: content/docs/tutorials/visualizing_metrics_using_grafana.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: Visualizing metrics using Grafana
+sort_rank: 4
+---
+
+# Visualizing metrics.
+
+In this tutorial we will create a simple dashboard using [Grafana](https://github.com/grafana/grafana) to visualize the `ping_request_count` metric that we instrumented in the previous tutorial.
```
Comment (hdost) [inline content/docs/tutorials/visualizing_metrics_using_grafana.md:8]: For "previous tutorial" i realized we should probably have a link.

**Review by hdost — CHANGES_REQUESTED** [review]
Just a couple small touches, that I didn't notice before. 
Looks great though 😊

**Comment by RichiH** [comment]
Thanks both of you!

**Pull request merged**